### PR TITLE
runescape-client: Rename CipherBuffer to PacketBuffer

### PR DESCRIPTION
**Pull request by kylestev**

### CODE REVIEW

File: runescape-client/src/main/java/Client.java [hunk 1]
```diff
@@ -227,10 +227,10 @@
    )
    static int field345;
    @ObfuscatedName("ch")
-   static CipherBuffer field346;
+   static PacketBuffer loginBuffer;
    @ObfuscatedName("ca")
-   @Export("secretCipherBuffer2")
-   static CipherBuffer secretCipherBuffer2;
+   @Export("ingressBuffer")
+   static PacketBuffer ingressBuffer;
```
Comment (kylestev) [inline runescape-client/src/main/java/Client.java]: Alternative name: `incomingPacketBuffer`

File: runescape-client/src/main/java/Client.java [hunk 2]
```diff
@@ -793,8 +793,8 @@
    @ObfuscatedName("ky")
    static int[] field493;
    @ObfuscatedName("cp")
-   @Export("secretCipherBuffer1")
-   static CipherBuffer secretCipherBuffer1;
+   @Export("egressBuffer")
+   static PacketBuffer egressBuffer;
```
Comment (kylestev) [inline runescape-client/src/main/java/Client.java]: Alternative name: `outgoingPacketBuffer`

File: runescape-client/src/main/java/Client.java [hunk 3]
```diff
@@ -227,10 +227,10 @@
    )
    static int field345;
    @ObfuscatedName("ch")
-   static CipherBuffer field346;
+   static PacketBuffer loginBuffer;
```
Comment (Adam-) [inline runescape-client/src/main/java/Client.java]: This should be exported

Comment (kylestev) [inline runescape-client/src/main/java/Client.java]: Good catch. Fixed in my latest commit.

**Comment by Adam-** [comment]
This needs to be redone post-update.

**Comment by kylestev** [comment]
Sure thing.

**Pull request merged**